geomet_data_registry/layer/base.py

```diff
@@ -50,6 +50,10 @@ def __init__(self, provider_def):
         self.model = None
         self.model_run = None
         self.wx_variable = None
+        self.layer_name = None
+        self.date_ = None
+        self.file_dict = None
+        self.new_key_store = False
```

Comment:
do we need `new_key_store`? If yes, why?

Reply:
indeed it is not needed anymore. It was there because at first I was gonna use another approach. 

Tagging as resolved.